roles/java/tasks/keystores.yml

```diff
@@ -82,7 +82,7 @@
         pkcs12_path: >-
           {{ download_location }}/{{ item.path | basename }}
         pkcs12_password: "{{ item.pass }}"
-        cert_alias: "{{ inventory_hostname }}"
+        cert_alias: "{{ inventory_hostname | ansible.builtin.hash }}"
```

Comment:
Are you sure this is really needed? 
Checking with keyttool it looks like there is no restriction on the certificate alias in the keystore:

```sh
$ keytool -list -keystore /tmp/k.jks

Type de fichier de clés : JKS
Fournisseur de fichier de clés : SUN

Votre fichier de clés d'accès contient 1 entrée

some/host_that*is$weird, 15 nov. 2023, trustedCertEntry, 
Empreinte du certificat (SHA-256) : 81:2E:C2:BE:ED:A9:CF:28:08:21:6B:EC:13:F8:86:71:A8:FE:04:A2:14:91:B9:C0:7F:12:C9:1F:72:2B:41:ED
```

Reply:
This summer, I read somewhere on the Oracle website that special characters weren't supported for aliases but I must say I didn't try myself. Based on your example, it doesn't look like it's a problem...